modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/engine/genesis.kt

```diff
@@ -34,13 +36,27 @@ open class AliceHas100XorAndPermissionToBurn : DefaultGenesis() {
     }
 }
 
+open class FewAssets : DefaultGenesis() {
+    companion object {
+        val DEFINITION_ID1 = DefinitionId("name1", DEFAULT_DOMAIN_NAME)
```

Comment:
Can you give it another name please? Not like SOMETHING1. You can give it metanames FOO, BAR, BAZ. Or use real tokens used in Sora project: XOR and VAL